File: app/src/main/java/com/github/oryanmat/trellowidget/widget/AlarmReceiver.java
package com.github.oryanmat.trellowidget.widget;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class AlarmReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        TrelloWidgetProvider.Companion.updateWidgets(context);
        TrelloWidgetProvider.Companion.updateWidgetsData(context);
    }
}

File: app/src/main/kotlin/com/github/oryanmat/trellowidget/widget/AlarmReceiver.kt
package com.github.oryanmat.trellowidget.widget

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class AlarmReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        context.updateWidgets()
        context.updateWidgetsData()
    }
}